native/src/routes/__tests__/SearchModal.spec.tsx

```diff
@@ -85,4 +85,16 @@ describe('SearchModal', () => {
       }
     })
   })
+
+  it('should show nothing found if there are no search results', () => {
+    const { getByText, getByPlaceholderText } = render(
+      <ThemeProvider theme={buildConfig().lightTheme}>
+        <SearchModal {...props} />
+      </ThemeProvider>
+    )
+
+    fireEvent.changeText(getByPlaceholderText('searchPlaceholder'), 'invalid query')
+
+    expect(getByText('search:nothingFound')).toBeTruthy()
```

Comment:
Could you also add a test for the CitySelector?

Reply:
Done